### DISCUSSION TITLE
2.3.0 linux arm64版本安装后不能启动

### DISCUSSION THREAD

**Discussion by xiaozihexuer** [lyswhut/lx-music-desktop#1476] [Q&A]
如题，我的问题是可以安装，但是启动不了，终端运行显示如下：
/opt/lx-music-desktop/lx-music-desktop
/opt/lx-music-desktop/lx-music-desktop: symbol lookup error: /opt/lx-music-desktop/lx-music-desktop: undefined symbol: gbm_bo_map
系统是统信uos 1050 华为擎云l410 专业版，用google搜了下没看到类似问题，求修复，谢谢

**Comment by Eastinliang** [comment]
> As in the title, my problem is that it can be installed, but it cannot be started. The terminal display is as follows: /opt/lx-music-desktop/lx-music-desktop /opt/lx-music-desktop/lx-music-desktop: symbol lookup error: /opt/lx-music-desktop/lx-music-desktop: undefined symbol: gb The m_bo_map system is Tongxin uos 1050 Huawei Qingyun l410 Professional Edition. I searched with Google and found no similar problems. Please fix it, thank you

https://github.com/lyswhut/lx-music-desktop/releases/download/v2.3.0/lx-music-desktop-v2.3.0-win_x64-green.7z

**Comment by lyswhut** [comment]
重复：<https://github.com/lyswhut/lx-music-desktop/issues/1215>
